inductiva/simulation/command.py

```diff
@@ -0,0 +1,19 @@
+"""Classe for construction of simulator commands."""
+
+
+class Command():
+    """Base class for construction of simulator commands."""
+
+    def __init__(self, **kwargs):
```

Comment:
Yes, i did like this so to be a template classe that can be overwritten by other classes for more specific commands, for example the GROMACSCOmmands where the method_name is always necessary. Do you think we can do it any other way?

Reply:
I think we will only use this when we have several `method_name`'s. Otherwise, I won't need a pipeline.

In general, I will assume that this allows me to run the simulator commands without requiring the "gmx" which is standard for gromacs. For openfoam, I will just assume this are the commands that the binaries allow.

Reply:
I think `method_name` should be part of the attributes of the base class. A command only exists if it has a name.